docs/build/messages/custom.mdx

```diff
@@ -86,6 +86,12 @@ export class ContentTypeMultiplyNumberCodec
     )}`
     // return undefined to indicate that this content type should not be displayed if it's not supported by a client
   }
+
+
+  // This method is optional and can be used to determine if the content type should have a push notification
```

Comment:
I don't think this is optional. It's an override method so for any codec it must be set. Integrators creating new codecs should air on the side of true. 
Should push should be set to true if the codec you are using is interoperable and should be displayed on other apps and clients.
The receiving clients push notification has to handle the should push field otherwise it will be sent by default.